plug.vim

```diff
@@ -1625,7 +1625,7 @@ class Plugin(object):
     self.write(Action.DONE, self.name, result[-1:])
 
   def repo_uri(self):
-    cmd = 'git rev-parse --abbrev-ref HEAD 2>&1 && git config -f .git/config remote.origin.url'
+    cmd = 'git rev-parse --abbrev-ref HEAD 2>&1 && unset GIT_CONFIG && git config remote.origin.url'
```

Comment:
Should the unset be done before the first `git rev-parse`, I wonder? Could an errant `GIT_CONFIG` somehow change how `--abbrev-ref` determines the branch name?

Reply:
Re my comment, if that's the case, note that that would be an _existing_ bug rather than a problem introduced by this PR. I think this PR could be merged as is.